styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -267,6 +267,9 @@ void concurrentlyIndexActiveWorkflowInstances() {
               return null;
             }
 
+            LOG.debug("adding the missing entry {} to {}", expectedEntry,
```

Comment:
Are we running with debug logging enabled?

Perhaps the log message should mention workflow instances and index?

I'd be tempted to track this using a metric but I guess we won't be keeping this around for more than a week or so.

Reply:
Yeah we do now. Will add more info to the message. Didn’t want to bother metrics as you said.